addOns/retire/src/main/java/org/zaproxy/addon/retire/model/Extractors.java

```diff
@@ -124,11 +127,11 @@ static String fixPattern(String inPattern) {
                 // [a-z]{8}
                 goodPattern = goodPattern.replaceAll("\\{\\}", "\\\\{\\\\}");
             }
-            if (goodPattern.endsWith(VERSION_SUB_PATTERN + ")")) {
-                // If the pattern ends with a version sub pattern, then artificially bound it with a
-                // whitespace check
-                goodPattern = goodPattern + "\\s";
-            }
+            //  if (goodPattern.endsWith(VERSION_SUB_PATTERN + ")")) {
+            // // If the pattern ends with a version sub pattern, then artificially bound it with a
+            // // whitespace check
+            // goodPattern = goodPattern + "\\s";
+            // }
```

Comment:
I'm happy to completely remove this. It seems I had previously added it to address other false positives, however all of those test cases seem to still work with today's revision.

https://github.com/zaproxy/zap-extensions/pull/3156/files